java-optional/src/solutions/java/none/cvg/optional/TestSolution1OptionalCreationAndFetchingValues.java

```diff
@@ -159,12 +159,12 @@ public void getValueFromOptionalForNonNullValue() {
          * DONE:
          *  Replace the "null" to get the Optional has a null value.
          *  Verify that the call throws a NoSuchElementException
-         *  Check API: java.util.Optional.isPresent()
```

Comment:
Good catch!

This comment should read: "Check API: java.util.Optional.get()".

We have not yet introduced the orElseThrow method at this point. 

The solutions in the kata need not be the most optimal, but are designed to teach concepts as people start learning. The second exercise (Test2**.java) goes into conditionals.